ui/js/service/http.service.ts

```diff
@@ -64,6 +64,15 @@ export class HTTPService {
     );
   }
 
+  authHeader(options?: any): string {
+    if (localStorage.getItem("jwt")) {
```

Comment:
I believe the order here is a bit off. We should be checking `options.email && options.password` first before checking `localStorage.getItem("jwt")`. If the token expires before the user logout then it will always try to create a bearer token.

Reply:
👍